h2o-hadoop/h2o-mapreduce-generic/src/main/java/water/hadoop/h2odriver.java

```diff
@@ -1352,6 +1359,15 @@ private int run2(String[] args) throws Exception {
               "};"
       );
       addMapperConf(conf, "-login_conf", "login.conf", krbConfData);
+    } else if (pamLogin) {
+      // Use default PAM configuration file
+      final byte[] pamConfData = StringUtils.bytesOf(
+              "pamloginmodule {\n" +
+                      "     de.codedo.jaas.PamLoginModule required\n" +
+                      "     service = h2o;\n" +
+                      "};"
+      );
+      addMapperConf(conf, "-login_conf", "login.conf", pamConfData);
     }
```

Comment:
That makes sense. The issue is that we require login_conf in H2O for the other login modules. I didn't want to introduce an exception for PAM.